readthedocs/doc_builder/environments.py

```diff
@@ -145,6 +143,13 @@ def run(self):
             env_paths.insert(0, self.bin_path)
         environment['PATH'] = ':'.join(env_paths)
 
+        log.info(
+            "Running build command.",
+            command=self.get_command(),
+            cwd=self.cwd,
+            environment=environment,
```

Comment:
I did a bit of debugging here, and we're calling docker `exec_create` with `environment`, but it doesn't appear to support it.... https://docker-py.readthedocs.io/en/1.7.1/api/#exec_create

🙃